### DISCUSSION TITLE
Page rules don't work properly for Guest group (from my understanding)

### DISCUSSION THREAD

**Discussion by AlexJFox** [requarks/wiki#6896] [Error / Bug Report] [answered]
I have a page at `/admin/debug/`
It is tagged with the tag `debug` (and no other tags)

The global read:pages permission is enabled for the Guest group and the page rules read as follows:
![image](https://github.com/requarks/wiki/assets/17463323/c5578441-86d6-4ef5-95e8-c22c74766f08)

As I understand it, the tag rule should override the path regex rule making the page visible, but it is not working this way. I would appreciate any advice on what I'm doing wrong. Thanks!

**Comment by NGPixel** [comment, marked as answer]
No, a deny will always override allow in the same path specificity. Tags are not more specific. You cannot use tags to override that as anyone can set any tags on pages, essentially giving yourself access.
